Casks/p/pingid.rb

```diff
@@ -8,8 +8,8 @@
   homepage "https://www.pingidentity.com/"
 
   livecheck do
-    url :url
-    strategy :extract_plist
+    url "https://docs.pingidentity.com/api/khub/maps/v6hqiVhowIDI52XNglkuRw/pages"
+    regex(/Desktop\s*app\s*v?(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/Desktop\s+app\s+v?(\d+(?:\.\d+)+)/i)
```

For what it's worth, I tend to use `\s+` to match expected whitespace (e.g., the spaces in "Desktop app 1.7.5") and `\s*` when dealing with potentially optional whitespace or trying to preemptively account for unexpected whitespace (e.g., in regexes like `>\s*something\s*<`).

Using `\s*` instead typically isn't an issue but it could technically lead to unexpected matches in some contexts. In this case, it's more about representing our intention, which is to match text like "Desktop app 1.7.5", not something like "Desktopapp1.7.5".

Reply:
Makes sense, thanks!